### PR TITLE
Create BB_RemotiveCloud

### PR DESCRIPTION
**Pull request by emil-dot-d**
Proposed new FEDERATE BB

### CODE REVIEW

File: BB-CEST/BB_RemotiveCloud [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+RemotiveCloud
+
+BB Tags(s) 
+BB-CEST; BB-CSC-TC
+
+Functional Cluster(s) (optional)	
+Diagnostics (could also be used for feeding vehicle data to emulators and other prototyping tools)
+
+Layer	
+Cloud
```
Comment (madri2) [inline BB-CEST/BB_RemotiveCloud:11]: Is it a plain Cloud Application or there are also Cloud Middleware Services included which can by for example re-used by other Cloud App Developers?

File: BB-CEST/BB_RemotiveCloud [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Known Implementation(s)	
+https://cloud.remotivelabs.com/ 
+
+ID (unique name)	
+
+Description	
+RemotiveCloud - for sharing vehicle recordings, both for debugging purposes as well as for prototyping (feed data into e.g. ProtoPie and Android AAOS / AOSP emulators). The recordings can be made with the RemotiveBroker or some other logging devices following standard formats.   
+Rationale	RemotiveCloud enables an easy way of sharing recordings - upload to cloud and share with relevant people as suppliers and service providers to have a common place for vehicle network data (CAN etc.) for debugging and 3rd party application developers.      
+
+Governance Applicable S-BB(s)
```
Comment (madri2) [inline BB-CEST/BB_RemotiveCloud:22]: Are there any S-BBs under consideration like: EU General Data Protection Regulation (GDPR)?

**Review by madri2 — CHANGES_REQUESTED** [review]
Hi @emil-dot-d I'm not allowed to make changes to the file.
There are two comments from my side, more general questions and:
Please rename the file to BB_RemoteCloud.md
Please insert a # before each line that is a heading.

**Comment by LS0205** [comment]
> Hi @emil-dot-d I'm not allowed to make changes to the file. There are two comments from my side, more general questions and: Please rename the file to BB_RemoteCloud.md Please insert a # before each line that is a heading.

I renamed and added headings for you @emil-dot-d. Please take a look at @madri2 follow-up questions.

File: BB-CEST/BB_RemotiveCloud (hunk 2 above)
Comment (AnotherDaniel) [reply BB-CEST/BB_RemotiveCloud]: IMO, GPRD is not a building block but a large regulation that implies a number of specific functionalities (BBs) - these might become part of the FEDERATE catalog, but on the other hand, they are not at all automotive-specific, so I think they are out of SDV scope.

**Review by AnotherDaniel — APPROVED** [review]
I will approve this PR for now, to enable the project to proceed with straightening out the current batch of PRs.

However, I strongly think that this is not a BB but a larger solution (composed of individual building blocks), and as such has no place in the BB catalog. Also see #36

File: BB-CEST/BB_RemotiveCloud.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# RemotiveCloud
+
+## BB Tags(s) 
+BB-CEST; BB-CSC-TC
+
+## Functional Cluster
+Diagnostics (could also be used for feeding vehicle data to emulators and other prototyping tools)
+
+## Layer	
+Cloud
```
Comment (LS0205) [inline BB-CEST/BB_RemotiveCloud.md:10]: there is no "Cloud layer" -> options are HWLayer, OSLayer, MWLayer, AppLayer

**Review by LS0205 — CHANGES_REQUESTED** [review]
The BB has the FC "Diagnostics". @emil-dot-d please put it in a folder named "Diagnostics"

File: BB-CEST/BB_RemotiveCloud (hunk 2 above)
Comment (madri2) [reply BB-CEST/BB_RemotiveCloud]: For such Regulations we have defined the S-BB Tag, Supporting BBs which are regulations, standards ... 
If for example a vehicle recording (data set) is stored in the cloud and the vehicle owner can be identified on the basis of the stored data, ....

File: BB-CEST/BB_RemotiveCloud (hunk 1 above)
Comment (madri2) [reply BB-CEST/BB_RemotiveCloud]: I think we can resolve this discussion.

**Review by madri2 — APPROVED** [review]
fine

**Pull request closed without merging**